OutOfSchool/OutOfSchool.WebApi/Controllers/ChildrenController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OutOfSchool.Services;
+using OutOfSchool.Services.Models;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Services;
+using OutOfSchool.WebApi.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OutOfSchool.WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]/[action]")]
+    [Authorize]
+    public class ChildrenController : ControllerBase
+    {
+        private IChildService childService;
+
+        public ChildrenController(IChildService childService)
+        {
+            this.childService = childService;
```

Comment:
please be consistent with naming
if you choose naming without underscore, use this approach in other classes as well